libraries/AP_Mount/AP_Mount_config.h

```diff
@@ -14,7 +14,7 @@
 #endif
 
 #ifndef HAL_MOUNT_GREMSY_ENABLED
-#define HAL_MOUNT_GREMSY_ENABLED AP_MOUNT_BACKEND_DEFAULT_ENABLED && BOARD_FLASH_SIZE > 1024
+#define HAL_MOUNT_GREMSY_ENABLED AP_MOUNT_BACKEND_DEFAULT_ENABLED && HAL_GCS_ENABLED && BOARD_FLASH_SIZE > 1024
```

Comment:
I guess this means that the gremsy is not enabled if mavlink is not enabled?

Reply:
Yes.  It's a mavlink-based gimbal :-)